src/locale/locales/es/messages.po

```diff
@@ -1023,28 +1023,28 @@ msgstr "Libros"
 
 #: src/components/FeedInterstitials.tsx:350
 msgid "Browse more accounts on the Explore page"
-msgstr ""
+msgstr "Explorar más cuentas en la página Explore"
 
 #: src/components/FeedInterstitials.tsx:483
 msgid "Browse more feeds on the Explore page"
-msgstr ""
+msgstr "Explore más alimentos en la página de explorar"
```

Comment:
Uhm... What? XD `alimentos`? Either we keep `feeds` or try to play it around like this:
https://x.com/RAEinforma/status/1309481926751858688

Same for all instances of `feed/s` when is a `social-media feed`

Reply:
resolved with "Revisar más feeds en la página de exploración"